web/client/localConfig.json

```diff
@@ -363,7 +363,11 @@
                 "name": "Measure",
                 "cfg": {
                   "defaultOptions": {
-                    "showAddAsAnnotation": true
+                    "showAddAsAnnotation": true,
+                    "trueBearing":{
```

Comment:
You can remove this from localConfig ( it's optional and it  will be documented)